lib/hooks/userspace/hook-redis.js

```diff
@@ -124,7 +156,11 @@ module.exports = function(version_, agent_) {
         shimmer.wrap(redis, 'createClient', createClientWrap);
       },
       unpatch: function(redis) {
-        shimmer.unwrap(redis.RedisClient.prototype, 'send_command');
+        if (semver.satisfies(version_, '<2.6')) {
+          shimmer.unwrap(redis.RedisClient.prototype, 'send_command');
+        } else {
+          shimmer.unwrap(redis.RedisClient.prototype, 'internal_send_command');
+        }
```

Comment:
nit: This is pretty much a copy of the wrap code above. As it gets longer, it would be increasingly worrisome lest we have syntactic typos that diverge behaviour between wrap/unwrap. (For example if the range comparison were to be subtly different due to a typo...). Not sure if this is worth it yet.